install_files/ansible-base/roles/ossec_server/templates/send_encrypted_alarm.sh

```diff
@@ -1,4 +1,3 @@
 #!/bin/bash
 
-/usr/bin/formail -I "" | /usr/bin/gpg --homedir /var/ossec/.gnupg --trust-model always -ear "{{ ossec_gpg_fpr }}" | /usr/bin/mail -s "$SUBJECT" {{ ossec_alert_email }}
-
+/usr/bin/formail -I "" | /usr/bin/gpg --homedir /var/ossec/.gnupg --trust-model always -ear "{{ ossec_gpg_fpr }}" | /usr/bin/mail -s "$(echo $SUBJECT | sed 's/{{ app_ip }}//g' )" {{ ossec_alert_email }}
```

Comment:
Is the `app_ip` guaranteed to be the only IP that ever appears in an OSSEC subject line? From skimming my backlog of OSSEC test alerts, that appears to be the cause, but I don't know if it it guaranteed to be. If not, I'd prefer to just use a simple regex to strip any IP addresses from the subject line.